### PR TITLE
overlord/tests: fix panic in managers test

### PR DESCRIPTION
**Pull request by stolowski**
Mock snap-confine profile of mocked core snaps used in the tests to avoid errors in setup-profiles tasks.

### CODE REVIEW

**Comment by zyga** [comment]
I just started a run of this on my machine. Let's see if I can hit the panic.

File: overlord/managers_test.go [hunk 1]
```diff
@@ -113,6 +115,16 @@ func (ms *mgrsSuite) SetUpTest(c *C) {
 	err := os.MkdirAll(filepath.Dir(dirs.SnapStateFile), 0755)
 	c.Assert(err, IsNil)
 
+	// mock snap-confine profile inside core snap rev 1 and 30, required by setup-profiles tasks.
```
Comment (mvo5) [inline overlord/managers_test.go]: Why 1-30 ? Do we really use so many mocked snap-confines?

Comment (stolowski) [reply overlord/managers_test.go]: No, we use only 2  revisions: "1" and "30". It's not a 1..30 range :)

Comment (mvo5) [reply overlord/managers_test.go]: Uhhh - silly me - sorry for the noise.

**Comment by mvo5** [comment]
This is superseded by https://github.com/snapcore/snapd/pull/6227 now - yes?

**Comment by zyga** [comment]
I think we should close this and instead merge https://github.com/snapcore/snapd/pull/6226 with the rationale given there in the commit message.

**Pull request closed without merging**